app/Store.php

```diff
@@ -6,5 +6,9 @@
 
 class Store extends Model
 {
-    //
+    function reviews()
+    {
+        // リレーション StoreモデルにReviewは属する
+        return $this -> belongsTo('App\Review');
```

Comment:
Store : Review = 1 : 多
ですよね？ 👀 

１つのお店が複数のレビュー（口コミ）をもっているはずなので
ここは
```suggestion
        return $this -> hasMany('App\Review');
```
じゃないですかね？
（Review modelの方も逆）

Reply:
修正しましたが、すでに中川さんが上げてマージ済みですので削除します。